cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunProfilerTask.java

```diff
@@ -52,6 +52,7 @@ public class RunProfilerTask implements Task {
     private final PrintStream err;
     private static final String JAVA_OPTS = "JAVA_OPTS";
     private static final String CURRENT_DIR_KEY = "current.dir";
+    private final Path targetOutputPath = Paths.get(System.getProperty(USER_DIR));
```

Comment:
Shall we make this constant?
private static final Path TARGET_OUTPUT_PATH = Paths.get(System.getProperty(USER_DIR));